samples/net/ocpp/Kconfig

```diff
@@ -0,0 +1,22 @@
+# Config options for OCPP sample application
+
+# Copyright (c) 2024 Linumiz
+# SPDX-License-Identifier: Apache-2.0
+
+mainmenu "OCPP sample application"
+config OCPP_SERVER
```

Comment:
I would prefer what we namespace the sample specific Kconfig options properly, otherwise it is difficult to say what option is "generic" and what is sample specific. So for networking samples we have used `NET_SAMPLE` prefix for this.
So please change the options to be called `NET_SAMPLE_OCPP_SERVER`, `NET_SAMPLE_OCPP_PORT`, `NET_SAMPLE_SNTP_SERVER` etc.